ValidicCSharp/Request/UrlBuilder.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace ValidicCSharp.Request
+{
+    using System.Linq;
+
+    public class UrlBuilder
+    {
+        public static string FromCommand(Command command)
```

Comment:
we will be adding more tests, this code is not new, it was moved from the Command class.